packages/valory/connections/ledger/ledger_dispatcher.py

```diff
@@ -338,6 +346,46 @@ async def get_transaction_receipt(
             )
         return response
 
+    def send_signed_transactions(
+        self,
+        api: LedgerApi,
+        message: LedgerApiMessage,
+        dialogue: LedgerApiDialogue,
+    ) -> LedgerApiMessage:
+        """
+        Send the request 'send_signed_transactions'.
+
+        :param api: the API object.
+        :param message: the Ledger API message
+        :param dialogue: the Ledger API dialogue
+        :return: response Ledger API message
+        """
+        try:
+            signed_transactions = message.signed_transactions.signed_transactions
+            transaction_digests = api.send_signed_transactions(
+                signed_transactions,
+                raise_on_try=True,
+                **message.kwargs.body,
+            )
+        except Exception as e:  # pylint: disable=broad-except  # pragma: nocover
+            return self.get_error_message(e, api, message, dialogue)
+
+        if transaction_digests is None:  # pragma: nocover
+            return self.get_error_message(
+                ValueError("No transaction_digest returned"), api, message, dialogue
+            )
+
+        return cast(
+            LedgerApiMessage,
+            dialogue.reply(
+                performative=LedgerApiMessage.Performative.TRANSACTION_DIGESTS,
```

Comment:
Addressed https://github.com/valory-xyz/open-aea/pull/594#discussion_r1142885883 , @0xArdi please confirm this is correct

Reply:
Looks good